pkg/instancegroups/instancegroups.go

```diff
@@ -34,6 +37,8 @@ import (
 	"k8s.io/kops/upup/pkg/fi"
 )
 
+const rollingUpdateTaintKey = "kops.k8s.io/rolling-update"
```

Comment:
Perhaps `.../rolling-update-in-progress`?

Reply:
I'd use something like `/scheduled-for-update`

Reply:
i'm on board with that

Reply:
Changed